alerts/apps_alerts.libsonnet

```diff
@@ -235,7 +235,7 @@
             'for': '15m',
           },
           {
-            alert: 'KubeJobCompletion',
+            alert: 'KubeJobIncomplete',
```

Comment:
```suggestion
            alert: 'KubeJobNotCompleted',
```

`Incomplete` sounds like the Job is missing something.

Reply:
I like `KubeJobNotCompleted` too